htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HxRequest.java

```diff
@@ -6,9 +6,41 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation for mapping htmx requests onto specific handler methods.
+ * Annotation for mapping htmx requests onto specific handler method.
  */
 @Target({ElementType.TYPE, ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface HxRequest {
+
+    /**
+     * Restricts the mapping to the {@code id} of a specific target element.
+     *
+     * @see <a href="https://htmx.org/reference/#request_headers">HX-Target</a>
+     */
+    String target() default "";
+
+    /**
+     * Restricts the mapping to the {@code id} of a specific triggered element.
+     *
+     * @see <a href="https://htmx.org/reference/#request_headers">HX-Trigger</a>
+     */
+    String triggerId() default "";
+
+    /**
+     * Restricts the mapping to the {@code name} of a specific triggered element.
+     *
+     * @see <a href="https://htmx.org/reference/#request_headers">HX-Trigger-Name</a>
+     */
+    String triggerName() default "";
+
+    /**
+     * Restricts the mapping to the {@code id}, if any, or to the {@code name} of a specific triggered element.
+     * <p>
+     * If only one of them should be restricted, use the {@link #triggerId()} or {@link #triggerName()} parameters instead.
```

Comment:
Is this actually the case? I believe the `value` proposal was there as a convenience, but it should be ok I think to just use `@HxRequest("someId")` and `someId` is never used as a name. I see `value` just as an alias to either `triggerId` or `triggerName`. If you want to be explicit use `triggerId` or `triggerName`, but if it is clear enough for you and your project, you can use `value`. Or do you see it differently?

Reply:
No. I'll adjust the javadoc.